assets/js/modules/analytics-4/datastore/accounts.js

```diff
@@ -43,24 +43,41 @@ import { createFetchStore } from '../../../googlesitekit/data/create-fetch-store
 import { actions as errorStoreActions } from '../../../googlesitekit/data/create-error-store';
 import { createValidatedAction } from '../../../googlesitekit/data/utils';
 import { isValidAccountSelection } from '../utils/validation';
+import { caseInsensitiveListSort } from '../../../util/case-insensitive-sort';
+import { appendAccountID, appendPropertyAndAccountIds } from '../utils/account';
 
 const { receiveError, clearError, clearErrors } = errorStoreActions;
 
 const fetchGetAccountSummariesStore = createFetchStore( {
 	baseName: 'getAccountSummaries',
-	controlCallback() {
+	controlCallback( params ) {
+		const { pageToken = '' } = params || {};
```

Comment:
This is often how we do it in other instances I believe.
```suggestion
	controlCallback( { pageToken } ) {
```